testing/e2e_dbaas_test.go

```diff
@@ -99,39 +101,36 @@ func TestDbaasKafkaLifecycle(t *testing.T) {
 	require.NoError(t, err)
 	t.Logf("Created Kafka Topic with name: %s", topicName)
 
-	// List topics
-	resp, err = c.CallTool(ctx, mcp.CallToolRequest{
-		Params: mcp.CallToolParams{
-			Name: "db-cluster-list-topics",
-			Arguments: map[string]interface{}{
-				"id":       cluster.ID,
-				"page":     "1",
-				"per_page": 10,
+	require.Eventually(t, func() bool {
+		resp, err := c.CallTool(ctx, mcp.CallToolRequest{
+			Params: mcp.CallToolParams{
+				Name: "db-cluster-list-topics",
+				Arguments: map[string]interface{}{
+					"id":       cluster.ID,
+					"page":     "1",
+					"per_page": 10,
+				},
```

Comment:
In case of number of topics > 10, we'll have topics leftover ? I think we are paginating but not handling if there are more than 1 page ?